### DISCUSSION TITLE
Need to keep current tab open

### DISCUSSION THREAD

**Discussion by buffon-georges** [cypress-io/cypress#18790] [Questions and Help]
Hello everyone, 

I know that Cypress isn't designed to support multiple tabs but I have a particular use case for which i cannot find any issue. 

Here is my scenario : i need to change my password on a site. To do so, I'm asked to go seek for a OTP (One Time Password) that was sent to my inbox. Once I have to code, I must come back to that site and enter it. If I close the current tab, then the OTP will expire. 

Did you guys already face that issue? Is there any workaround for the scenario that I'm facing ? 
(main domain -> second domain -> main domain)

Thanks in advance.

**Comment by Glamor23** [comment]
Yes
